### DISCUSSION TITLE
email.it non va da qualche settimana

### DISCUSSION THREAD

**Discussion by marcobartman** [matteobaccan/html2pop3#182] [General]
Ciao! Purtroppo gli account email.it hanno smesso di funzionare da qualche settimana, e restituiscono "invalid password" anche se ovviamente le credenziali sono corrette. User inviata correttamente come "<utente>@email.it", e crittografia non abilitata, come richiesto. Problema riscontrato con tutte le caselle email.it (ne ho più di una, su PC diversi). Riesci ad aggiornare? Sarebbe top! Grazie mille come sempre! ;-)

**Comment by matteobaccan** [comment]
Ciao

non ricordo se ti ho già scritto che la versione 3.0.7 risolve il problema

https://github.com/matteobaccan/html2pop3/releases/tag/3.0.7

ciao
matteo

**Reply by marcobartman** [reply]
Grazie mille, l'ho scaricata appena uscita e funziona! ;-)
